Sources/WAU/Winget-AutoUpdate/Winget-Upgrade.ps1

```diff
@@ -254,7 +255,8 @@ if (Test-Network) {
                     New-Item "$WorkingDir\logs\error.txt" -Value "Blacklist doesn't exist in GPO" -Force
                     Exit 1
                 }
-                $toSkip = $toSkip.Data
+                $toSkip = $toSkip
```

Comment:
Same ?

Reply:
yep that'd be the same, I've removed that whole line now